src/impl/project/diff/profileDiff.ts

```diff
@@ -0,0 +1,151 @@
+import ProfileRetriever from "../../metadata/retriever/profileRetriever";
```

Comment:
See my comment on ProfileRetreiver, it doesnt make sense for a profile Diff to import ProfileRetriever, lets bring the common writing element to shared or even another impl which is all about writing a profile

Reply:
@genoud  Once this is done.. this is good to go